GameData/MOARdV/MAS_ASET/FlagIndicator/MAS_FI_Landed.cfg

```diff
@@ -54,7 +54,7 @@
 			blend = true
 			activeColor = COLOR_ASET_SWITCHER_NAME_POSITIVECOLOR
 			passiveColor = COLOR_ASET_SWITCHER_NAME_ZEROCOLOR
-			text = CONTACT
+			text = LANDED
```

Comment:
Flag Indicators are configured through the propFlagIndicator.xml file, so the corresponding entry there needs update (I don't have an objection to this change - just letting you know that there's an extra step involved to prevent it being overwritten)

Reply:
done